tests/KO/275.lp

```diff
@@ -0,0 +1,8 @@
+symbol const A : TYPE
+symbol const a : A
+symbol f : (A ⇒ A) ⇒ A
+rule f (λ x, &X &Y) → a
+ and f (&X &Y) → a
```

Comment:
This line is not tested because the previous one will fail I guess. It might be a good idea to have several test files.